app/pathogen/arbovirus/visualizations/page.tsx

```diff
@@ -23,26 +24,29 @@ export default function VisualizationsPage() {
   }
 
   return (
-    <div className="flex-col flex w-screen h-screen overflow-y-scroll">
-      <div className="flex pt-4">
-        <h3 className="w-full text-center text-lg">
-          {visualizationInformation.displayName}
-        </h3>
-        <button 
-          className="pr-8"
-          hidden={!referrerRoute || !isSafeReferrerLink(referrerRoute)}
-          onClick={() => {
-            if(referrerRoute && isSafeReferrerLink(referrerRoute)) {
-              router.push(referrerRoute)
-            }
-          }}
-          aria-label="Close Visualization"
-        >
-          <X />
-        </button>
-      </div>
-      <div className={cn("flex-1", visualizationInformation.classNameWhenFullscreen)}>
-        {visualizationInformation.renderVisualization()}
+    <div className="w-screen overflow-y-hidden grid grid-cols-12 grid-rows-2 -my-4 -ml-4 h-full-screen">
```

Comment:
Same problem as the `wip` page. I have to fight the layout of the pages we're going to remove. The negative margin can be removed when we make the `wip` page the main dashboard.